client/message/messages.js

```diff
@@ -68,21 +66,5 @@ Messages.propTypes = {
   messages: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
 }
 
-const byDateDescending = R.comparator((d1, d2) => d1.sent > d2.sent)
-
-const getGroupMessages = (state, {params}) => {
-  const groupUid = params.groupUid;
-  return {
-    messages: R.sort(byDateDescending, R.values(getMessagesByGroup(state, groupUid))),
-  }
-}
-
-const getStoryMessages = (state, {params}) => {
-  const storyUid = params.storyUid;
-  return {
-    messages: R.sort(byDateDescending, R.values(getMessagesByStory(state, storyUid))),
-  }
-}
-
 export const GroupMessages = connect(getGroupMessages)(Messages)
```

Comment:
Now that these getters are outside of this file I mentioned a few changes (commented in `reducers/index.js`.

Here I'd like to split it out so we can extend it, (it's a little strange to use getters in place of `stateToProps`

Also, note how cool it is we can completely re-use the Messages component just by switching out `stateToProps`, it's pretty nifty :)

Here's the simple version
```
export const GroupMessages = connect(state=> ({messages: getGroupMessages})(Messages)
export const GroupMessages = connect(state=> ({messages: getStoryMessages})(Messages)
```